rpcs3/Emu/Cell/Modules/cellSpudll.cpp

```diff
@@ -1,22 +1,47 @@
 #include "stdafx.h"
 #include "Emu/Cell/PPUModule.h"
 
-namespace vm { using namespace ps3; }
+#include "cellSpudll.h"
 
 logs::channel cellSpudll("cellSpudll", logs::level::notice);
 
-s32 cellSpudllGetImageSize(vm::ptr<u32> psize, vm::cptr<void> so_elf, vm::cptr<struct CellSpudllHandleConfig> config)
+s32 cellSpudllGetImageSize(vm::ptr<u32> psize, vm::cptr<void> so_elf, vm::cptr<CellSpudllHandleConfig> config)
 {
-	fmt::throw_exception("Unimplemented" HERE);
+	cellSpudll.todo("cellSpudllGetImageSize(psize=*0x%x, so_elf=*0x%x, config=*0x%x)", psize, so_elf, config);
+
+	if (!psize || !so_elf)
+	{
+		return CELL_SPUDLL_ERROR_NULL_POINTER;
+	}
+
+	// todo
+
+	return CELL_OK;
 }
 
-s32 cellSpudllHandleConfigSetDefaultValues(vm::ptr<struct CellSpudllHandleConfig> config)
+s32 cellSpudllHandleConfigSetDefaultValues(vm::ptr<CellSpudllHandleConfig> config)
 {
-	fmt::throw_exception("Unimplemented" HERE);
+	cellSpudll.trace("cellSpudllHandleConfigSetDefaultValues(config=*0x%x)", config);
+
+	if (!config)
+	{
+		return CELL_SPUDLL_ERROR_NULL_POINTER;
+	}
+
+	config->mode = 0;
+	config->dmaTag = 0;
+	config->numMaxReferred = 16;
+	config->unresolvedSymbolValueForFunc = vm::null;
+	config->unresolvedSymbolValueForObject = vm::null;
+	config->unresolvedSymbolValueForOther = vm::null;
```

Comment:
Not sure about these three lines above. I have wanted to use NULL but found vm::null instead. Is that correct ?